demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatMyCarWorthPage/MyCarValuePage.java

```diff
@@ -0,0 +1,86 @@
+package com.example.test.pages.KBBMOOCProject.WhatMyCarWorthPage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.example.test.pages.KBBMOOCProject.KbbBasePage;
+
+public class MyCarValuePage extends KbbBasePage {
+
+   public MyCarValuePage(WebDriver driver) {
+      super(driver);
+      // TODO Auto-generated constructor stub
+   }
+
+   WebElement btnMakeModel = keyword.findWebElementByXpath("//div[text()='Make/Model']");
+   WebElement btnLicensePlate = keyword.findWebElementByXpath("//div[text()='License Plate']");
+
+   public void clickOnMakeModelCheckBoxInMyCarValuePage() throws InterruptedException {
+      keyword.clickThenWait(btnMakeModel);
+   }
+
+   public void clickOnLicensePlateCheckBoxInMyCarValuePage() throws InterruptedException {
+      keyword.clickThenWait(btnLicensePlate);
+   }
+
+   public void clickOnYearDropDownInMyCarValuePage() throws InterruptedException {
+      WebElement btnYear = keyword.findWebElementByXpath("//div[@class='year']");
+      keyword.clickThenWait(btnYear);
+   }
+
+   public void clickToChooseaYearInYearDropDownInMyCarValuePage() throws InterruptedException {
+      WebElement optYear = keyword.findWebElementByXpath("//div[@class='year']//option[@value= '2020']");
+      keyword.clickThenWait(optYear);
+   }
+
+   public void clickOnMakeDropDownInMyCarValuePage() throws InterruptedException {
+      WebElement btnMake = keyword.findWebElementByXpath("//div[@class='make']");
```

Comment:
should write the web element locator in separate line then call into a method/function